arrow-udf-js/Cargo.toml

```diff
@@ -14,7 +14,9 @@ arrow-array.workspace = true
 arrow-buffer.workspace = true
 arrow-schema.workspace = true
 atomic-time = "0.1"
-rquickjs = { version = "0.6", features = ["array-buffer", "parallel"] }
+rquickjs = { version = "0.6", features = ["array-buffer", "parallel", "futures"] }
```

Comment:
We are locked in v0.6.x because `rquickjs` has moved to `quickjs-ng` since 0.7, which doesn't support `BigDecimal` 🥵 

https://github.com/DelSkayn/rquickjs/commit/f14a8f8f4b12a1ef83f742794b958c34a40e514e#diff-5ec820142c45eb6726a2231c4e7825e12b5d4c24963c01ccc92de2512d2dc99eL80